docs/library_usage.md

```diff
@@ -27,6 +27,16 @@ This section contain some help for those brave enough to use its internals as a
     --8<-- "read_two_sensors.out"
     ```
 
+=== "sensor stream (low-level API)"
+
+    ``` python
+    --8<-- "sensor_stream.py"
+    ```
+
+    ```
+    --8<-- "sensor_stream.out"
+    ```
```

Comment:
This example needs to be updated to the current PR.
It should refer to `read_one_sample.py` instead of `sensor_stram.py`

Reply:
Oops, thanks for spotting - amended to fix 👍.